docs/content/en/reference/multi-cluster/_index.md

```diff
@@ -169,34 +167,4 @@ Here is a brief summary of what the script does:
 * Extract the CA cert
 * Create a kubeconfig using the token and cert
 * Create a secret for the kubeconfig in the control plane custer
-* Create a ClientConfig in the control plane cluster that references the secret.
-
-Suppose we have two clusters with context names `kind-k8ssandra-0` and `kind-k8ssandra-1` and the control plane is running
-in `kind-k8ssandra-0`. We want to create a ClientConfig for `kind-k8ssandra-1`. This can be accomplished by running the
-following:
-
-```sh
-./create-clientconfig.sh \
-    --namespace             k8ssandra-operator \
-    --src-kubeconfig        ./kubeconfigs/k8ssandra-1 \
-    --dest-kubeconfig       ./kubeconfigs/k8ssandra-0 
-```
-
-The above assumes that each cluster has its configuration stored in distinct files. If you have one single kubeconfig
-for both clusters, then you can create the ClientConfig as follows:
-
-```sh
-./create-clientconfig.sh \
-    --namespace             k8ssandra-operator \
-    --src-kubeconfig        ./kubeconfig \ 
-    --dest-kubeconfig       ./kubeconfig \
-    --src-context           kind-k8ssandra-1 \ 
-    --dest-context          kind-k8ssandra-0
-```
-
-Here `./kubeconfig` refers to the kubeconfig file containing configurations to access both contexts `kind-k8ssandra-0`
-and `kind-k8ssandra-1`.
-
-You can specify the namespace where the secret and ClientConfig are created with the `--namespace` option.
-
-**Note:** Remember to restart the operator in the control plane cluster after creating the ClientConfig.
\ No newline at end of file
+* Create a ClientConfig in the control plane cluster that references the secret.
```

Comment:
**Suggestion:**

```suggestion
* Create a ClientConfig in the control plane cluster that references the secret.

See [here]({{< relref "/tasks/data-plane-registration" >}}) for more information on the dataplane registration procedure.
```

Reply:
What's the difference here, I must be blind..?

Reply:
I added line 172 ;)

Reply:
Oh I get it, thanks :)